configure-windows

```diff
@@ -7,16 +7,8 @@
 
 
 export prefix=C:/ocamlmgw64
-while : ; do
```

Comment:
I'm not sure, this is a good change. Would you like to explain this, @Et7f3 ?

Reply:
We throw away all arguments except --prefix so instead of this destructive analysis give all args to configure script directly.

If in the original flags we have --prefix we would then have --prefix=default-path --prefix=new-path other arguments